recipe/yum_requirements.txt

```diff
@@ -0,0 +1,16 @@
+liburiparser-devel
```

Comment:
Welcome to the naming nightmare. Things might exist but with a different name or they are parts of a package with a different name.

You will need a lot `yum search` to sort that out. Sorry...